src/components/Footer/footer.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from 'next/link';
+import React from 'react';
+import { footerLink } from '../../../static';
+Link
+
+function Footer() {
+  return (
+    <footer className='bg-[#F4F6FE] mt-32 flex justify-between p-8'>
```

Comment:
Similarly here, I'd move the margin top 32 to the parent component, in this case the Layout. Something along this lines:

```
      <Header  />
      <main className="pb-32" /* or "mb-32" */>{children}</main>
      <Footer />
```